src/Thinktecture.Relay.Server.Persistence.EntityFrameworkCore/StatisticsRepository.cs

```diff
@@ -165,9 +165,9 @@ public async Task CloseConnectionAsync(string connectionId)
 		}
 
 		/// <inheritdoc />
-		public async Task CleanUpConnectionsAsync(TimeSpan oldestToKeep)
+		public async Task CleanUpConnectionsAsync(TimeSpan maxAge)
 		{
-			var filterTime = DateTime.UtcNow - oldestToKeep;
+			var filterTime = DateTime.UtcNow - maxAge;
 
 			_logger.LogDebug("Cleaning up statistics storage: Deleting all connections that have not been updated since {OldestOriginToKeep}.", filterTime);
```

Comment:
```suggestion
			_logger.LogDebug("Cleaning up statistics storage by deleting all connections that have no activity since {ConnectionLastActivity}.", filterTime);
```